components/console-backend-service/internal/domain/servicecatalogaddons/listener/clusteraddonsconfiguration.go

```diff
@@ -5,21 +5,21 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/kyma-project/kyma/components/console-backend-service/internal/gqlschema"
-	v1 "k8s.io/api/core/v1"
+	"github.com/kyma-project/kyma/components/helm-broker/pkg/apis/addons/v1alpha1"
 )
 
-//go:generate mockery -name=gqlAddonsConfigurationConverter -output=automock -outpkg=automock -case=underscore
-type gqlAddonsConfigurationConverter interface {
-	ToGQL(item *v1.ConfigMap) *gqlschema.AddonsConfiguration
+//go:generate mockery -name=gqlClusterAddonsConfigurationConverter -output=automock -outpkg=automock -case=underscore
+type gqlClusterAddonsConfigurationConverter interface {
+	ToGQL(item *v1alpha1.ClusterAddonsConfiguration) *gqlschema.AddonsConfiguration
 }
 
 type AddonsConfiguration struct {
```

Comment:
It's a valid point, but we can take care of that when we will introduce the `AddonsConfiguration` kind. Right now I do not want to retrigger the CI jobs because of that renaming :D